docs/testing-styleguide.md

```diff
@@ -0,0 +1,90 @@
+# Testing styleguide
+
+We use ["Futurelearn style" acceptance tests](https://about.futurelearn.com/blog/how-we-write-readable-feature-tests-with-rspec).
+
+## Rules
```

Comment:
Wondering if there should be something about newlines and readability? It's very common that we have multiple given, when, then blocks in the same scenario, if they didn't have a newline between each I would find it hard to read. For example:

```ruby
scenario 'User does a thing' do
  given_i_am_signed_in
  when_i_press_a_button
  then_something_should_happen
  given_i_am_on_this_page
  when_i_do_something
  then_something_else_should_happen
end
```

Vs

```ruby
scenario 'User does a thing' do
  given_i_am_signed_in
  when_i_press_a_button
  then_something_should_happen

  given_i_am_on_this_page
  when_i_do_something
  then_something_else_should_happen
end
```

Reply:
Added!